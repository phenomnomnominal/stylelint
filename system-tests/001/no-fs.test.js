'use strict';

const browser = require('../../lib/browser');
const stylint = require('../../lib');
const { caseConfig, caseCode, prepForSnapshot } = require('../systemTestUtils');

const CASE_NUMBER = '001';

describe('no-fs - valid sanitize.css and their config', () => {
	let browserResult;
	let stylintResult;

	beforeAll(async () => {
		const code = await caseCode(CASE_NUMBER);
		const config = await caseConfig(CASE_NUMBER);

		stylintResult = prepForSnapshot(await stylint.lint({ code, config }));
		browserResult = prepForSnapshot(await browser.lint({ code, config }));
	});

	it('standalone', async () => {
		expect(stylintResult).toMatchSnapshot();
	}, 10000);

	it('standalone and browser return equal results', async () => {
		expect(browserResult).toEqual(stylintResult);
	}, 10000);
});
