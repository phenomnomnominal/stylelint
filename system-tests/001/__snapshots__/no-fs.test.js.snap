// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-fs - valid sanitize.css and their config standalone 1`] = `
Object {
  "errored": false,
  "output": Array [
    Object {
      "deprecations": Array [],
      "errored": false,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [],
    },
  ],
  "results": Array [
    Object {
      "deprecations": Array [],
      "errored": false,
      "ignored": undefined,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [],
    },
  ],
}
`;
